script/CreateProposals-epoch24.s.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.23;
+
+import { Script, console2 } from "../lib/forge-std/src/Script.sol";
+
+import { IGovernor } from "../lib/ttg/src/abstract/interfaces/IGovernor.sol";
+import { IStandardGovernor } from "../lib/ttg/src/interfaces/IStandardGovernor.sol";
+
+import { DeployBase } from "./DeployBase.sol";
+
+contract CreateProposals is Script, DeployBase {
+    // address internal constant _STANDARD_GOVERNOR = 0xB024aC5a7c6bC92fbACc8C3387E628a07e1Da016; // Mainnet Standard Governor
+    address internal constant _STANDARD_GOVERNOR = 0x89C867D0a4B2d4Adc6DFD00d2f153D7794Bf9Fef; // Testnet Standard Governor
+
+    bytes32 internal constant _SOLANA_EARNERS_PREFIX = "solana-earners";
+
+    // Proposal descriptions
+    string internal constant _WM_SOLANA_VAULT_DESC =
+        "# Add the Wrapped M (wM) Solana M Vault PDA (base58: 8vtsGdu4ErjK2skhV7FfPQwXdae6myWjgWJ8gRMnXi2K, hex: 0x75d03cbc601c7143e02620783c2d31eee8c51897f080da17be44aeb6e3298490) to the 'solana-earners' list in preparation for the deployment of M and wM on Solana. This PDA will be the signer on the wM Earn Program and will custody M that has been wrapped.\n\nSince Solana addresses are longer (32 bytes) than EVM addresses (20 bytes), we must use the more generic `setKey(bytes32 key, bytes32 value)` function on the Registrar. The key is calculated as `keccak256(abi.encodePacked(bytes32('solana-earners'),bytes32(0x75d03cbc601c7143e02620783c2d31eee8c51897f080da17be44aeb6e3298490))). The value is just a boolean flag, so we set it to 1 (meaning the address is a member of the list).";
```

Comment:
is the intent to add first 2 sentences to title vs 1st one? 
\n\n split title from actual description